common/scala/src/main/resources/application.conf

```diff
@@ -114,6 +114,14 @@ whisk {
         actions-ddoc = "whisks.v2.1.0"
         activations-ddoc = "whisks.v2.1.0"
         activations-filter-ddoc = "whisks-filters.v2.1.0"
+
+        # Size limit for inlined attachments. Attachments having size less than this would
+        # be inlined with there content encoded in attachmentName
+        max-inline-size = 16 k
+
+        # Chunk sized for converting source of bytes to ByteString as part of attachment
+        # upload flow
+        chunk-size = 8 k
```

Comment:
Value of 8k is choosed based on default used by [StreamConverters.fromInputStream][1] which is used by `DocumentFactory` to create a `Source[ByteString,_]` from `InputStream` currently

[1]: https://github.com/akka/akka/blob/master/akka-stream/src/main/scala/akka/stream/scaladsl/StreamConverters.scala#L48